### PR TITLE
Added renewed checklist, risk-dto.ts nad tests in risk-score-api.spec.ts

### PR DESCRIPTION
**Pull request by mon-bene**

### CODE REVIEW

File: tests/dto/risk-score-api.spec.ts [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+  riskDto.loanPeriod = 9
+  const response = await request.post(
+    'https://backend.tallinn-learning.ee/api/loan-calc/decision',
+    {
+      data: riskDto,
+    },
+  )
+  console.log('response status:', response.status())
+  console.log('response body:', await response.json())
+  expect.soft(response.status()).toBe(StatusCodes.OK)
```
Comment (vladimir-tl) [inline tests/dto/risk-score-api.spec.ts:41]: Verify a response code, and **all fields** in the response using soft asserts

File: tests/dto/risk-score-api.spec.ts [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+  const response = await request.post(
+    'https://backend.tallinn-learning.ee/api/loan-calc/decision',
+    {
+      data: riskDto,
+    },
+  )
+
+  console.log('response status:', response.status())
+  console.log('response body:', await response.json())
+  expect.soft(response.status()).toBe(StatusCodes.OK)
```
Comment (vladimir-tl) [inline tests/dto/risk-score-api.spec.ts:89]: we should check whether decision is negative

**Review by vladimir-tl — CHANGES_REQUESTED** [review]
good job and need some changes

File: tests/dto/risk-score-api.spec.ts (hunk 1 above)
Comment (mon-bene) [reply tests/dto/risk-score-api.spec.ts:41]: added to all tests all fields

File: tests/dto/risk-score-api.spec.ts (hunk 2 above)
Comment (mon-bene) [reply tests/dto/risk-score-api.spec.ts:89]: used the same logic with soft

**Review by vladimir-tl — APPROVED** [review]

**Pull request merged**